msteams-platform/graph-api/proactive-bots-&-messages/graph-proactive-bots-&-messages.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Is it a good idea to put a & in a filename?

Reply:
Probably not 😏